memory/combat_manager.py

```diff
@@ -39,13 +40,24 @@ def __init__(self):
         self.spell_locks = []
 
 
+class CombatCharacter(Enum):
+    NONE = "None"
+    Zale = "Zale"
+    Valere = "Valere"
+    Garl = "Garl"
+    _SPOILERS = "_SPOILERS"
+
+
 class CombatPlayer:
     def __init__(self, params=dict):
```

Comment:
Not sure what the `params=dict` will do here. In either case, `params` seems to not be used?

Reply:
vestages of old bad code - removing